http.c

```diff
@@ -142,6 +140,10 @@ static http_curlopt settable_curlopts[] = {
 	{ "CURLOPT_TCP_KEEPALIVE", CURLOPT_TCP_KEEPALIVE, CURLOPT_LONG, false },
 	{ "CURLOPT_TCP_KEEPIDLE", CURLOPT_TCP_KEEPIDLE, CURLOPT_LONG, false },
 #endif
+#if LIBCURL_VERSION_NUM >= 0x072500 /* 7.37.0 */
```

Comment:
I see these at 7.52.0 in my table. Did you find a different version number somewhere else?